README.md

```diff
@@ -157,6 +156,95 @@ Working in progress....
+
+![Reset Password page](./docs/images/wireframes/set-new-password-page.png)
+
+### Search page
+
+The search bar enables users to refine their product search using keywords or by selecting from the categories in the drop-down menu. The resulting matches will be presented on the search results page.
+
+![Search page](./docs/images/wireframes/search-page.png)
+
+### Product details page
+
+Clicking on a product image or name will direct users to the Product Details page, where they can find more information about the product and add it to their cart if they wish to make a purchase.
+
+![Product Details page](./docs/images/wireframes/product-details-page.png)
+
+### Shopping cart page
+
+When users click the cart icon in the header, they will be taken to the Cart page. Here, they can view all the products they have added and can choose to proceed with their purchase by clicking the 'Proceed to checkout' button or continue shopping by clicking the 'Continue shopping' button.
+
+![Shopping Cart Page](./docs/images/wireframes/shopping-cart-page.png)
+
+### Checkout page
+
+Users intent on proceeding to payment must log in or register an account to be taken to the Checkout page after selecting the 'Proceed to Checkout' button on the cart page. They are required to provide delivery address details and credit card information, and must review and confirm the order summary to finalize the payment process.
+
+![Checkout Page](./docs/images/wireframes/checkout-page.png)
+
+### User account page
+
+Authenticated users have access to their personal accounts where they can view their profiles and purchase history. Additionally, they have the option to change their current password in this section.
+
+![User Account Page](./docs/images/wireframes/user-account-page.png)
+
+### Admin dashboard page
+
+The admin dashboard is organized into four tabbed pages for streamlined navigation:
+
+- Summary: This tab allows the admin to quickly glance at the business's key metrics, such as the total number of products, orders, and registered users, all consolidated on a single summary screen.
+- Products: This crucial section is the heart of product management within the admin dashboard, administrators have the authority to oversee the entire product list, introduce new ones, edit existing product details, or remove products from the listing.
+- Categories: Similar to the Products tab, this section empowers the admin to manage product categories. Here, the admin can view all existing categories, add new categories, update information on current categories, or remove categories as needed. This ensures that the product catalog remains well-organized and easy to navigate.
+- Users: This tab enables the admin to review a list of all registered users on the platform.
+- Orders: In this section, the admin has the capability to overview the list of all orders, and update the statuses of these orders.

```

Comment:
same here, can you change `overview the list` to `view the list`?

also, we can keep the sentence simple: `the admin has the capability` to `the admin is able to` 

and no need plurals for status because there's only 1 status per order.